src/BaGetter.Azure/Storage/BlobStorageService.cs

```diff
@@ -22,7 +22,7 @@ public async Task<Stream> GetAsync(string path, CancellationToken cancellationTo
         {
             return await _container
                 .GetBlockBlobReference(path)
-                .OpenReadAsync(cancellationToken);
+                .OpenReadAsync();
```

Comment:
Why not pass the token?

Reply:
The api changed: They discarded the CT as single argument /facepalm